disvae/models/losses.py

```diff
@@ -235,8 +238,24 @@ def __call__(self, data, model, optimizer, storer):
             vae_loss = rec_loss + kl_loss + self.gamma * tc_loss
         else:
             # return vae loss without mutual information term
+            # change latent dist to torch.tensor (could probably avoid this)
+            latent_dist = torch.stack((latent_dist[0], latent_dist[1]), dim=2)
+            # calculate log p(z)
+            prior_params = torch.zeros(half_batch_size, latent_dist.size(1), 2)
+            logpz = log_density_normal(latent_sample1, prior_params, half_batch_size,
```

Comment:
Maybe log_pz is more readable here? Just a personal preference

Reply:
Going to leave as is for now.